index.html

```diff
@@ -16,8 +16,9 @@
         <div class="row">
           <!-- ko foreach: calculators -->
           <div class="col-lg-6 col-md-4 col-sm-6 col-xs-12 calculator">

```

Comment:
`Each widget would be the same size as well.` from https://github.com/SMH110/widgets/issues/44

It looks to me as though the To Do List and the Calculator would be different sizes at some resolutions